### ISSUE TITLE
Is "Conda Build" badage actually needed?

### ISSUE THREAD

**Issue by erogluorhan** [NCAR/geocat-comp#247]

**Comment by marodrig** [comment]
The Conda Build badge is different from the CI builds badge. Both badges have different an valuable information, and I think for the time being it's ok to have both.

**Comment by erogluorhan** [comment]
Yes, but when you click on the Conda Build badge, it links to [this](https://github.com/NCAR/geocat-comp/actions?query=workflow%3Abuild_test), which shows very old build_tests (the newest is from 2 years ago). 

So, even though the idea   of testing Conda builds is good, I think the implementation is not testing up-to-date stuff.

**Comment by marodrig** [comment]
@erogluorhan Got it. I'll take a closer look into it and update it.
